.github/workflows/main.yml

```diff
@@ -139,3 +139,38 @@ jobs:
         verbose: true
         fail_ci_if_error: true
         flags: python
+
+  helm-testing:
+    name: Lint & Test Helm chart
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
+        with:
+          fetch-depth: 0
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.8.1
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@main
```

Comment:
not caused by your change but we should probably refer to specific versions or tags, not `@main`

for increased stability + "security"